src/olympia/abuse/management/commands/migrate_abuse_reports_to_country_code.py

```diff
@@ -0,0 +1,19 @@
+from django.core.management.base import BaseCommand, CommandError
+
+from olympia.abuse.models import AbuseReport
+
+
+class Command(BaseCommand):
+    def setup_check(self):
+        value = AbuseReport.lookup_country_code_from_ip('1.1.1.1')
+
+        if not value:
+            raise CommandError('GeoIP lookups does not appear to be working.')
+
+    def handle(self, *args, **kwargs):
+        self.setup_check()
+
+        qs = AbuseReport.objects.filter(ip_address__isnull=False)
+        for report in qs:
+            value = AbuseReport.lookup_country_code_from_ip(report.ip_address)
+            report.update(country_code=value)
```

Comment:
This needs logging. I don't know how many abuse reports there are, but if there are many this may take some time?

Reply:
The number if known, just don't want to publicly share it - it might take some time but I think it'll be low enough that it can be done in one go. In any case, it's really just a bonus, there is no real need to do the migration, it's just nice to have.

Reply:
Added logging in e7b2b9e82ad1c744da7df7933a8bd5131db3b78a, and also stop loading useless fields and made the queryset avoid instances that already had a country code set